buildSrc/build.gradle.kts

```diff
@@ -29,6 +27,22 @@ dependencies {
 
     val ktlint_version = libs.versions.ktlint.version.get()
     implementation("org.jmailen.gradle:kotlinter-gradle:$ktlint_version")
+
+    implementation("io.ktor:ktor-server-default-headers:2.0.2")
```

Comment:
is it possible/needed to update this version with every new release?

Reply:
also, why not to use version catalog as with other dependencies in buildSrc build script?

Reply:
We can put `*` here, but I'm not sure why do we need this

Reply:
It will be actually good to migrate manually with IDE plugin

Reply:
> why not to use version catalog as with other dependencies in buildSrc build script?

we could, but it would be listed as ktor_version and confused with actual project version

Reply:
ktor_build_version or it can be named some else way :)

BTW IMO it will be better to extract this test server into additional composite build, and don't put it inside buildSrc